### PR TITLE
Added references to later versions of Windows

### PR DESCRIPTION
**Pull request by ld0614**
I've had an go at making the modifications suggested in #1736, I added all current OSs to the supported list, I wasn't sure of the exact linguistics of 'Windows Vista and above and Windows Server 2008 and above'.  Happy to modify as required 😄

### CODE REVIEW

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ ms.technology: identity-adds
 ---
 # Enabling Clients to Locate the Next Closest Domain Controller
 
->Applies To: Windows Server 2016, Windows Server 2012 R2, Windows Server 2012
+>Applies To: Windows Server 2019, Windows Server 2016, Windows Server 2012 R2, Windows Server 2012, Windows Server 2008 R2, Windows Server 2008
```
Comment (MicrosoftGuyJFlo) [inline WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: Please remove ", Windows Server 2008 R2, Windows Server 2008" as we no longer refer to them in the tag lines.

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md [hunk 2]
```diff
@@ -13,9 +13,9 @@ ms.technology: identity-adds
...
 
-If you have a domain controller that runs  Windows Server 2008  or  Windows Server 2008 R2 , you can make it possible for client computers that run Windows Vista, Windows 7,  Windows Server 2008 , or  Windows Server 2008 R2  to locate domain controllers more efficiently by enabling the **Try Next Closest Site** Group Policy setting. This setting improves the Domain Controller Locator (DC Locator) by helping to streamline network traffic, especially in large enterprises that have many branch offices and sites.  
+If you have a domain controller that runs Windows Server 2008 and above, you can make it possible for client computers that run Windows Vista and above or Windows Server 2008 and above to locate domain controllers more efficiently by enabling the **Try Next Closest Site** Group Policy setting. This setting improves the Domain Controller Locator (DC Locator) by helping to streamline network traffic, especially in large enterprises that have many branch offices and sites.
```
Comment (MicrosoftGuyJFlo) [inline WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: Windows Server 2008 and above > Windows Server 2008 or newer

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md [hunk 3]
```diff
@@ -35,9 +35,9 @@ If you enable the **Try Next Closest Site** setting, DC Locator uses the followi
 - If no domain controller is available in the same site, try to find a domain controller in the next closest site. A site is closer if it has a lower site-link cost than another site with a higher site-link cost.  
 - If no domain controller is available in the next closest site, try to find any domain controller in the domain.  
 
-By default, DC Locator does not consider any site that contains a read-only domain controller (RODC) when it determines the next closest site. In addition, because only  Windows Server 2008  and  Windows Server 2008 R2  domain controllers support the next closest site functionality, when the client gets a response from a domain controller that runs an earlier version of Windows Server, the DC Locator behavior is the same as when then setting is not enabled.  
+By default, DC Locator does not consider any site that contains a read-only domain controller (RODC) when it determines the next closest site. In addition, when the client gets a response from a domain controller that runs an version earlier than Windows Server 2008, the DC Locator behavior is the same as when then setting is not enabled.
```
Comment (MicrosoftGuyJFlo) [inline WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: that runs an version earlier than Windows Server 2008, > that runs a version earlier than Windows Server 2008,

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md [hunk 4]
```diff
@@ -35,9 +35,9 @@ If you enable the **Try Next Closest Site** setting, DC Locator uses the followi
...
+By default, DC Locator does not consider any site that contains a read-only domain controller (RODC) when it determines the next closest site. In addition, when the client gets a response from a domain controller that runs an version earlier than Windows Server 2008, the DC Locator behavior is the same as when then setting is not enabled.  
   
-For example, assume that a site topology has four sites with the site link values in the following illustration. In this example, all the domain controllers are writable domain controllers that run  Windows Server 2008  or  Windows Server 2008 R2 .  
+For example, assume that a site topology has four sites with the site link values in the following illustration. In this example, all the domain controllers are writable domain controllers that run  Windows Server 2008  or  above.
```
Comment (MicrosoftGuyJFlo) [inline WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: Windows Server 2008 or above > Windows Server 2008 or newer

**Comment by MicrosoftGuyJFlo** [comment]
#hold-off

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md (hunk 4 above)
Comment (ld0614) [reply WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: done

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md (hunk 1 above)
Comment (ld0614) [reply WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: While I personally don't agree with this considering they still have 18 months of support left, I've updated in line with policy

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md (hunk 2 above)
Comment (ld0614) [reply WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: done throughout the document

File: WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md (hunk 3 above)
Comment (ld0614) [reply WindowsServerDocs/identity/ad-ds/plan/Enabling-Clients-to-Locate-the-Next-Closest-Domain-Controller.md]: done

**Comment by MicrosoftGuyJFlo** [comment]
@ld0614 Thanks for making those changes! Looks great!

**Comment by MicrosoftGuyJFlo** [comment]
Resolves https://github.com/MicrosoftDocs/windowsserverdocs/issues/1735
Resolves https://github.com/MicrosoftDocs/windowsserverdocs/issues/1736

**Comment by MicrosoftGuyJFlo** [comment]
#sign-off

**Comment by MicrosoftGuyJFlo** [comment]
BTW @ld0614 congrats on your first contribution to WindowsServerDocs!

**Pull request merged**